src/app/components/dashboard/dashboard.component.ts

```diff
@@ -12,22 +12,41 @@ import { Score } from '../../models/score';
 })
 export class DashboardComponent implements OnInit {
 
+  _score: Score = {score: [0, 0]};
+
   constructor(private scoreService: ScoreService,
               private gameStateService: GameStateService,
               private snackBar: MatSnackBar) {}
 
   ngOnInit() {
-    this.scoreService.score().subscribe(score => {
-      if (!this.isZeroZero(score)) {
-        this.showToast('GOOOOAL! Score is now' + ' ' + score.score[0] + ' - ' + score.score[1], 5000);
-      }
+    this.scoreService.score(0).subscribe(score => {
+      this._score.score[0] = score;
+      this.showScoreToast();
+    });
+
+    this.scoreService.score(1).subscribe(score => {
+      this._score.score[1] = score;
+      this.showScoreToast();
```

Comment:
- On new game the score will be '0'
- On reverting goals the team's score will decrease (goal reverted = no goal shot)